lms/views/error.py

```diff
@@ -52,12 +51,16 @@ def error(self):
 
 
     @view_config(context=MissingLTILaunchParamError)
-    def missing_lti_launch_param_error(self):
+    @view_config(context=MissingLTIContentItemParamError)
```

Comment:
`MissingLTILaunchParamError` and `MissingLTIContentItemParamError` are both descendants `LTILaunchError`; you could probably do `@view_config(context=LTILaunchError)` and nothing else here and it would work (I _presume_ that's how Pyramid view config would work).